src/Octopurls/redirects.json

```diff
@@ -105,6 +105,7 @@
   "AuthAD": "http://docs.octopus.com/display/OD/Active+Directory+authentication",
   "AuthAzureAD": "https://octopus.com/docs/administration/authentication/authentication-providers/azure-ad-authentication",
   "AuthGoogleApps": "https://octopus.com/docs/administration/authentication/authentication-providers/googleapps-authentication",
+  "AuthOctopusID": "https://octopus.com/docs/administration/authentication/authentication-providers/octopusid-authentication",
```

Comment:
```suggestion
  "AuthOctopusID": "https://octopus.com/docs/administration/authentication/octopusid-authentication",
```

Reply:
Not sure about ☝️ ... Just looking at the directory structure from https://github.com/OctopusDeploy/docs/pull/508/files, I'm not sure if `authentication-providers/` is superfluous here?

Reply:
Well spotted. It turns out the other auth provider things are relying on a redirect that's in place, because they got moved. I'll change to make them all consistent